src/installation/guides/fde.md

```diff
@@ -1,132 +1,131 @@
+On legacy BIOS systems this command would use `/dev/vda1`.
 
-Are you sure? (Type uppercase yes): YES
-Enter passphrase:
-Verify passphrase:
+```
+# cryptsetup luksFormat --type luks1 /dev/vda2
 ```
 
-Once the volume is created, it needs to be opened. Replace `voidvm` with an
-appropriate name. Again, this will be `/dev/sda2` on EFI systems.
+Once the volume is created, it needs to be opened. `voidvm` is a example name
+you can replace it with a name of your choosing. Again, this will be `/dev/vda1` on legacy BIOS systems.
 
 ```
-# cryptsetup luksOpen /dev/sda1 voidvm
-Enter passphrase for /dev/sda1:
+# cryptsetup open /dev/vda2 voidvm
 ```
 
-Once the LUKS container is opened, create the LVM volume group using that
-partition.
+### LVM
+
+Once the LUKS partition is opened you can create a [LVM](https://en.wikipedia.org/wiki/Logical_Volume_Manager_(Linux)) volume group using that
+partition. 
+LVM is optional and may have its advantages or disadvantages depending on your system configuration.
+
+The volume group name is not required to be the same as the LUKS partition name chosen above, 
+But for consistency purposes it may be easier to use the same name for the volume group.
 
 ```
 # vgcreate voidvm /dev/mapper/voidvm
-  Volume group "voidvm" successfully created
 ```
 
 There should now be an empty volume group named `voidvm`.
 
-Next, logical volumes need to be created for the volume group. For this example,
-I chose 10G for `/`, 2G for `swap`, and will assign the rest to `/home`.
+Next, logical volumes need to be created for the volume group. 
+For this example, All of the available space is assigned to `/`.
+
+The name chosen using `--name` should be noted because it will be used when mounting the partition, For example, `/dev/voidvm/root`.
 
 ```
-# lvcreate --name root -L 10G voidvm
-  Logical volume "root" created.
-# lvcreate --name swap -L 2G voidvm
-  Logical volume "swap" created.
-# lvcreate --name home -l 100%FREE voidvm
-  Logical volume "home" created.
+# lvcreate --name root -l 100%FREE voidvm
 ```
 
-Next, create the filesystems. The example below uses XFS as a personal
-preference of the author. Any filesystem [supported by
+### Filesystem creation
+
+Next, create the filesystems. The example below uses Btrfs. Any filesystem [supported by
 GRUB](https://www.gnu.org/software/grub/manual/grub/grub.html#Features) will
 work.
 
+On systems using LVM the root filesystem would be created on `/dev/voidvm/root`.
+
+```
+# mkfs.btrfs /dev/mapper/voidvm
+```
+
+On UEFI systems the EFI partition also needs the filesystem created.
+
 ```
-# mkfs.xfs -L root /dev/voidvm/root
-meta-data=/dev/voidvm/root       isize=512    agcount=4, agsize=655360 blks
-...
-# mkfs.xfs -L home /dev/voidvm/home
-meta-data=/dev/voidvm/home       isize=512    agcount=4, agsize=2359040 blks
-...
-# mkswap /dev/voidvm/swap
-Setting up swapspace version 1, size = 2 GiB (2147479552 bytes)
+# mkfs.vfat -F32 /dev/vda1
 ```
```

Comment:
1. Im confused on this one, Are you proposing dropping LVM?

2. Following my guide as is i didnt really find this a problem, 
If you add Btrfs subvolumes this might be a problem, but i tested that and didnt see a problem, You should also remember that LVM is noted as optional and may have its advantages and disavantages, 

(2/3.) And Btrfs is noted as the example filesystem, nobody is forcing the user to use Btrfs

3. I changed the default filesystem to Btfs because it is the "modern" filesystem gaining widespread adoption, 
Using XFS makes this guide look really dated,

I think a solution here is to leave one as the example and note about a few other common filesystems 
(the most common would probably be Btrfs, ext4, F2FS and XFS, excluding ZFS due to it having a separate guide already).